rubocop/lib/rubocop.yml

```diff
@@ -0,0 +1,210 @@
+require: rubocop-rspec
+
+AllCops:
+  Exclude:
+    - bin/**/*
+
+# LINT
+
+Lint/AmbiguousBlockAssociation:
+  # `scope :name, -> (param) { ... }` is a valid syntax
+  Enabled: false
+
+Lint/AmbiguousOperator:
+  Enabled: true
+
+Lint/NonLocalExitFromIterator:
+  Enabled: false
+
+Lint/HandleExceptions:
+  Enabled: false
```

Comment:
so what?! Looks like you ignore exception which is not ok in general. It is better to disable the cop in special cases, and enforce by default